### ISSUE TITLE
cmake scripts fail with Core

### ISSUE THREAD

**Issue by HoffmannMichael** [CGAL/cgal#4275]
## Issue Details

The scripts cgal_create_cmake_file and cgal_create_CMakeLists do not work with Core. The simple example below fails to link, no matter whether I use "cgal_create_cmake_file" or "cgal_create_CMakeLists -c Core" (with or without "-s ..."). I believe the issue goes back to this change

https://github.com/CGAL/cgal/commit/0bd9088b5ac32da69025598e43d7f02270363079#diff-cfccb193f8a88a81f8b6f117554f90e1

which removed
```cmakd
include( ${CGAL_USE_FILE} )
include( CGAL_CreateSingleSourceCGALProgram )
```
and went public in CGAL-4.14.

## Source 
```c++
#include <CGAL/Exact_predicates_exact_constructions_kernel_with_sqrt.h>

int main() 
{
 CGAL::Exact_predicates_exact_constructions_kernel_with_sqrt::FT x = 0;
}
 ```
## Environment

* Operating system (Mac, Linux):
* Compiler: clang 1000.11.45.5, gcc 9.1.1
* Release or debug mode: any
* Specific flags used (if any):
* CGAL version: 4.14
* Boost version: n/a
* Other libraries versions if used (Eigen, TBB, etc.): n/a

**Comment by lrineau** [comment]
I confirm the issue. The culprit is commit https://github.com/CGAL/cgal/commit/a3ddaa132fb17cbda662d814d6df3e8966e76ccc#diff-90eb902b01bf244f8d02e5423ed8d974, introduced in CGAL-4.12. And then that bug was revealed when `UseCGAL.cmake` was not longer used. In that commit, I have replaced the link with `${CGAL_LIBRARIES}` by a link with `CGAL::CGAL` only, forgetting CGAL_Core, CGAL_ImageIO, and CGAL_Qt5.

Now that I have seen the error, I will fix `CGAL_CreateSingleSourceCGALProgram.cmake` in a pull-request.

**Comment by HoffmannMichael** [comment]
Thanks! That works for me.

**Comment by lrineau** [comment]
Fixed by PR #4280.
